en/docs/what-is-devant.md

```diff
@@ -1,18 +1,20 @@
 # What is Devant?
 
-Devant allows you to manage your own AI integration agents. Integrate any AI agent into the integrations you build in low-code and pro-code, and jump from siloed systems to intelligent digital experiences with Devant by WSO2—the AI iPaaS that your AI Agents can call “home”.
+Devant is a powerful IPaaS with first-class AI support. Integrate any AI agent into the integrations you build in low-code and pro-code, and jump from siloed systems to intelligent digital experiences with Devant by WSO2—the AI iPaaS that your AI Agents can call “home”.
 
 ## Key capabilities of Devant
 
-### Design for security, governance, and efficiency
-- **Domain-Driven Design**: Organize enterprise systems into domains, exposing managed APIs, events, and data through projects. Each project consists of components representing units of deployment.
-- **Application Security Design**: Leverage [cell-based architecture](https://github.com/wso2/reference-architecture/blob/master/reference-architecture-cell-based.md), where projects become governable units from development to deployment, ensuring managed access via APIs, events, and data.
-
 ### Efficient and effective development
-- **Code in Any Language**: Develop applications in your team's preferred programming language, harnessing existing skill sets to build powerful polyglot APIs, services, events, tasks, and more.
-- **Native Support for VS Code**: Seamlessly collaborate and manage code using VS Code in a feature-rich, visually-enhanced development environment.
+- **AI-Assisted Development**: Devant includes a widely adopted VS Code-based editor with an integrated AI copilot, which aids developers at every stage, from code generation to refactoring.
```

Comment:
Please replace `widely adopted`, it sounds like editor is `widely adopted` not VSCode. eg `developer friendly`. Btw VSCdoe is one word.